apps/nextjs-pages-router/src/pages/_document.tsx

```diff
@@ -1 +1,7 @@
-export { Document as default } from '@makeswift/runtime/next'
+import { Document } from '@makeswift/runtime/next'
+
+export default class MyDocument extends Document {
+  static defaultProps = {
+    appOrigin: process.env.MAKESWIFT_APP_ORIGIN,
```

Comment:
Required to make the pages router work w/ a local builder.

Reply:
This is interesting. How come we're setting it as a static property on `Document` vs passing it as a prop to a component or as a parameter to a Makeswift constructor

Reply:
I'm seeing now this is using the `defaultProps` functionality from React, not something custom we're making. I think in the past we'd read directly from `process.env.MAKESWIFT_APP_ORIGIN` when destructuring the props. Or was this simply not working before?

Reply:
`Document` is a class component, defining `appOrigin` in `defaultProps` is effectively passing it as a prop. That prop is passed through to `<PreviewModeScript>` here: https://github.com/makeswift/makeswift/pull/836/files#diff-2d843b17408d4e8bbb89a142b11260df343c89c7a2c715a2f5153e78d8c26ffeR41